### PR TITLE
LPS-193923

### PR DESCRIPTION
**Pull request by TamyrisTorres**
Ticket: https://liferay.atlassian.net/browse/LPS-193923

### CODE REVIEW

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJSPFieldSetGroupCheck.testjsp [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJSPFieldSetGroupCheck.testjsp:1]: @TamyrisTorres, please remove this blank line

Comment (TamyrisTorres) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJSPFieldSetGroupCheck.testjsp]: resolved

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
@TamyrisTorres, can you please rebase to fix the conflicts and resend to me?

Thanks

**Pull request closed without merging**